wallet/file-format.md

```diff
@@ -81,7 +81,7 @@ The following table summarize the format:
 | Version | Mandatory | Integer | Entire part of this specification version | 1 |
 | Nickname | Mandatory | String || "Savings" |
 | Address | Optional | String || "AU12..." |
-| Salt | Mandatory | Byte array | Salt for PBKDF2 (16 Bytes) | [57, 125, 102, 235, 118, 62, 21, 145, 126, 197, 242, 54, 145, 50, 178, 98] |
+| Salt | Mandatory | String | Salt for PBKDF2 (16 Bytes, base 64 encoded) | "iad1ELW64ouX5HfCUCglig==" |
```

Comment:
@damip here there is the same example with trailing '=='. Should we remove it? 

also, why unpad base64? (I understand that unpadding is simple remove the trailing =, is it right?)

Reply:
yes, the goal is to have an unambiguous specification. Also on the rust side we need exactly that